libs/create-langchain-integration/template/scripts/jest-setup-after-env.js

```diff
@@ -0,0 +1,3 @@
+import { awaitAllCallbacks } from "@langchain/core/callbacks/promises";
```

Comment:
It was throwing earlier for me because it was trying to import a file that didn't exist. file was likely copypasta from langchain, there it had access to the file via `import ... from "../src/callbacks/promises.js"` but once inside these integrations that import would fail.

Reply:
Ah I thought I saw it even with the `core` import but may have been mistaken